capsules/src/process_console.rs

```diff
@@ -170,6 +176,49 @@ impl Default for WriterState {
     }
 }
 
+#[derive(Copy, Clone)]
+pub struct Command {
+    buf: [u8; COMMAND_BUF_LEN],
+    len: usize,
+}
+impl Command {
+    pub fn new() -> Command {
+        Command {
+            buf: [0; COMMAND_BUF_LEN],
+            len: 0,
+        }
+    }
+
+    /// Fill the buffer with the provided data.
+    /// If the provided data's length is smaller than the buffer length,
+    /// the left over bytes are not modified due to '\0' termination.
+    pub fn fill(&mut self, buf: &[u8], terminator_idx: usize) {
```

Comment:
```suggestion
    pub fn fill(&mut self, buf: &[u8l COMMND_BUF_LEN], terminator_idx: usize) {
```

Please use a reference to the array.